lib/ansible/module_utils/network/meraki/meraki.py

```diff
@@ -188,14 +187,16 @@ def get_org_id(self, org_name):
                     return str(i['id'])
 
     def get_nets(self, org_name=None, org_id=None):
+        ''' Downloads all networks in an organization '''
         if org_name:
             org_id = self.get_org_id(org_name)
         path = self.construct_path('get_all', org_id=org_id)
         r = self.request(path, method='GET')
         return r
 
     def get_net(self, org_name, net_name, data=None):
-        ''' Return network information '''
+        ''' Return network information about a particular network '''
+        ''' TODO: Allow method to download data on its own '''
```

Comment:
Could you please follow PEP 257 for docstring guidelines?
In Python, we use single docstring in the beginning of methods, tripple double quotes, full sentence, trailing period. Optional paragraph of extended description after an empty line break.

The second todo-line here looks like a comment to me.